.github/ci.yml

```
@@ -0,0 +1,54 @@
+name: Node CI
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - 'v*'
+  pull_request:
+
+jobs:
+  test:
+    name:  "Tests"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: pnpm/action-setup@v2
+        with:
+          version: 8
+      - name: Install Dependencies
+        run: pnpm i --frozen-lockfile
```

Would be nice to cache the dependencies so that the CI runs faster and saves energy. This is the easies way. [Reference](https://github.com/wyvox/action-setup-pnpm?tab=readme-ov-file#wyvoxaction-setup-pnpm)

```suggestion
      - uses: wyvox/action-setup-pnpm@v3
```